ci/build.ps1

```diff
@@ -36,7 +36,8 @@ $scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh'
   Invoke-Expression $scoop
 }
 
-scoop install diffutils perl
+scoop install diffutils
+scoop install perl
```

Comment:
> ci/Appveyor: install perl separately

 What's the motivation ?

Reply:
Yes.
scoop can install perl but not diffutils because of the issue with msys2.